web/src/components/nomination/NominateForm.vue

```diff
@@ -83,10 +83,10 @@
     </v-row>
     <urls-input
       v-model="form.urls"
-      :fb-errors="urlErrors($v.form.urls.facebook)"
-      :linkedin-errors="urlErrors($v.form.urls.linkedin)"
-      :twitter-errors="urlErrors($v.form.urls.twitter)"
-      :website-errors="urlErrors($v.form.urls.website)"
+      :fb-errors="urlErrors($v.form.urls.facebook, 'facebook')"
```

Comment:
This looks awesome. In the future, we might consider replacing these with i18n keys.

Reply:
@ann-kilzer thanks for the comment! Can you explain more about how it would work with i18n keys here? 🤔